post-processor/vsphere/post-processor.go

```diff
@@ -135,11 +160,15 @@ func (p *PostProcessor) PostProcess(ui packer.Ui, artifact packer.Artifact) (pac
 			password,
 			"<password>",
 			-1))
-	cmd := exec.Command("ovftool", args...)
-	cmd.Stdout = os.Stdout
-	cmd.Stderr = os.Stderr
+
+	var out bytes.Buffer
+	cmd := exec.Command(ovftool, args...)
+	cmd.Stdout = &out
+	cmd.Stderr = &out
```

Comment:
Is there a reason to not forward ovftool output to stderr/out? I see you're capturing stderr below, but I would like to see these tee'd out to the os descriptors.

It's also possible `out` will contain junk, since it's being written to simultaneously by two outputs.

something like

```go
var errOut bytes.Buffer
cmd := exec.Command("ovftool", args...)
errWriter = io.MultiWriter(os.StdErr, errOut)
cmd.Stdout = os.Stdout
cmd.Stderr = errWriter
```

Reply:
The previous behavior did not put ovftool errors on stderr. It was very difficult to pinpoint the cause of failure. I think your edit is much of improvement. "ovftool" though should be ovftool (without quotes) for the patch to work. I agree with you.

Reply:
I like hooker's example above.